### PR TITLE
Update disaster recovery guide to reflect error message change

### PR DESCRIPTION
**Pull request by jackwaudby**
Recently in 5.7 we updated the error message when attempting to deallocate a server hosting a database with only 1 primary. This PR updates the disaster recovery guide to reflect this.

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-709.surge.sh

**Comment by jackwaudby** [comment]
[Update disaster recovery with new deallocate of single error message](https://trello.com/c/Jd5zzxgJ/625-update-disaster-recovery-with-new-deallocate-of-single-error-message)

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 1]
```diff
@@ -117,7 +117,7 @@ If *all* servers show health `AVAILABLE` and status `ENABLED` continue to xref:c
 . On each `UNAVAILABLE` server, run `CALL dbms.cluster.cordonServer("unavailable-server-id")`.
 . On each `CORDONED` server, run `DEALLOCATE DATABASES FROM SERVER cordoned-server-id`.
 . On each server that failed to deallocate with one of the following messages:
-.. `Could not deallocate server [server]. Can't move databases in single mode [database]`
+.. `Could not deallocate server [server]. Can't move databases with only one primary [database]`
```
Comment (AlexicaWright) [inline modules/ROOT/pages/clustering/disaster-recovery.adoc:120]:
```suggestion
.. `Could not deallocate server [server]. Can't move databases with only one primary [database].`
```
Isn't there supposed to be a full stop at the end?

**Review by AlexicaWright — COMMENTED** [review]
Just one question.

**Pull request merged**